dolphinscheduler-master/src/main/java/org/apache/dolphinscheduler/server/master/runner/WorkflowExecuteThreadPool.java

```diff
@@ -182,6 +185,18 @@ private void notifyProcessChanged(ProcessInstance finishProcessInstance) {
         }
     }
 
+    private void crossWorkflowParameterPassing(ProcessInstance finishProcessInstance, TaskInstance taskInstance) {
+        try {
+            MasterTaskExecuteRunnable masterTaskExecuteRunnable =
+                    MasterTaskExecuteRunnableHolder.getMasterTaskExecuteRunnable(taskInstance.getId());
+            masterTaskExecuteRunnable.getILogicTask().getTaskParameters()
+                    .setVarPool(finishProcessInstance.getVarPool());
+            log.info("Cross workflow parameter passing success, processInstanceId: {}, taskInstanceId");
```

Comment:
The processInstanceId and taskInstanceId Not filled.

Reply:
Sorry, I didn't check carefully
`log.info("Cross workflow parameter passing success, finishProcessInstanceId: {}, taskInstanceId: {}", finishProcessInstance.getId(), taskInstance.getId());`
is this ok?

Reply:
Yes. Please supplement other relevant logs.

Reply:
Is there anything else you think needs to be added to the log?

Reply:
Exception error log should add too.

Reply:
I have added it, thanks for your advice.